platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/internal/component/resolution/failure/ResolutionFailureHandlerIntegrationTest.groovy

```diff
@@ -371,24 +371,32 @@ class ResolutionFailureHandlerIntegrationTest extends AbstractIntegrationSpec {
         and: "Has error output"
         failure.assertHasDescription("Could not determine the dependencies of task ':forceResolution'.")
         failure.assertHasCause("Could not resolve all dependencies for configuration ':resolveMe'.")
-        assertFullMessageCorrect("""   > Found multiple transforms that can produce a variant of root project : with requested attributes:
+        assertFullMessageCorrect("""   > Found multiple transformation chains that produce a variant of 'root project :' with requested attributes:
        - color 'red'
        - shape 'round'
-     Found the following transforms:
-       - From 'configuration ':roundBlueLiquidElements'':
+     Found the following transformation chains:
+       - From configuration ':roundBlueLiquidElements':
            - With source attributes:
                - color 'blue'
                - shape 'round'
                - state 'liquid'
-           - Candidate transform(s):
-               - Transform 'BrokenTransform' producing attributes:
-                   - color 'red'
-                   - shape 'round'
-                   - state 'gas'
-               - Transform 'BrokenTransform' producing attributes:
-                   - color 'red'
-                   - shape 'round'
-                   - state 'solid'""")
+           - Candidate transformation chains:
+               - Transformation chain: 'BrokenTransform', producing attributes:
+                   - 'BrokenTransform':
```

Comment:
`producing attributes` does not seem like it should be here.

Previously, this error listed the final attributes of the resulting variant, after each transform step was applied in order. 

Now, we do not display these final attributes, as we now display the individual transform steps that get us to the final variant. I think it is fine that we don't display the final attributes, as we display the raw data that contributed to the previous final attributes. But, the `producing attributes` text is not longer needed

Reply:
Good catch, I missed this.  I'll remove it.